repository/database/verification.py

```diff
@@ -51,3 +53,29 @@ def get_role_ids(self) -> List[int]:
 
     def set_role_ids(self, roles: List[int]) -> None:
         self.role_ids = json.dumps(roles)
+
+    def update_rule(self) -> None:
+        session.merge(self)
+        session.commit()
+
+    def remove_rule(self) -> None:
+        session.query(DynamicVerifyDB).filter(DynamicVerifyDB.id == self.id).delete()
```

Comment:
This should work as well, but I have not tested it:
```suggestion
        session.delete(self)
```
https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.delete